assets/src/edit-story/components/canvas/framesLayer.js

```diff
@@ -60,6 +86,13 @@ function FramesLayer() {
             return <FrameElement key={id} element={{ id, ...rest }} />;
           })}
         <Selection />
+        {Boolean(draggingResource) && isDropSource(draggingResource.type) && (
+          <FrameSidebar>
+            <Hint>
+              {__('Drop targets are outlined in blue', 'web-stories')}
```

Comment:
If we're doing this, we should probably also include "Use cmd to disable drop targets".

Reply:
You mean to disable snapping?

Reply:
Right. Nevermind. I keep confusing these two because these both ideas use the same shortcut. So I hope that doesn't come back :)